scripts/run-ci-tests.sh

```diff
@@ -118,7 +118,7 @@ else
     if [[ "$DISABLE_UNIT_TESTS" != "true" ]]; then
 	if [[ -n "${TEST_CONFIG}" ]]; then
         	echo "Running ${TEST_CONFIG} tests"
-        	ctest -L ${TEST_CONFIG} "${CTEST_OPTIONS}"
+        	ctest -VV -L ${TEST_CONFIG}
```

Comment:
It looks like setting CTEST_OPTIONS in travis.yml wasn't working because it was in quotes and treating multiple arguments as a single unrecognized argument.

```suggestion
        	ctest -L ${TEST_CONFIG} ${CTEST_OPTIONS}
```